README.md

```diff
@@ -15,22 +15,42 @@ With the release of AndroidX, Android Support is now considered deprecated and w
 
 ## Binding Policies
 
-- This repository binds over 100 AndroidX libraries that are published to [NuGet.org](https://nuget.org). The full package list can be found in [config.json](config.json).
+- This repository binds over 200 (2022-09) AndroidX libraries that are published to [NuGet.org](https://nuget.org). The full 
+  package list can be found in [config.json](config.json).
 - AndroidX Java artifacts come from [Google's Maven Respository](https://maven.google.com/web/index.html#).
+- dependency Maven artifacts come from different maven repositories mostly:
+  - [Google's Maven Respository](https://maven.google.com/web/index.html#) and
+  - [Maven/Sonatype Central](https://repo1.maven.org/maven2/)
 - Google's release notes for AndroidX libraries are available [here](https://developer.android.com/jetpack/androidx/versions/stable-channel).
-- The major/minor/patch version numbers mirror the AndroidX library version. For example, the NuGet `Xamarin.AndroidX.Core 1.3.2.1` binds version `1.3.2` of the AndroidX library `androidx.core:core`.
+- The major/minor/patch version numbers mirror the AndroidX library version. For example, the NuGet `Xamarin.AndroidX.Core 1.3.2.1` 
+  binds version `1.3.2` of the AndroidX library `androidx.core:core`.
   - The revision version number is used when a new NuGet needs to be built but the AndroidX library has not been updated.
-- We endeavor to release updated NuGets within a few weeks after new AndroidX releases, however large changes occasionally require more time.
+- We endeavor to release updated NuGets within a few weeks after new AndroidX releases, however large changes occasionally require 
+  more time.
 - In general, we do not bind pre-release libraries. As their API is not stable yet, it results in too much rework.
 
+### Details
+
+Full list of maven artifact to NuGet mappings:
+
+[./docs/artifact-list.md](./docs/artifact-list.md)
+
+Full list of maven artifact with versions to NuGet mappings with versions:
+
+[./docs/artifact-list-with-versions.md](./docs/artifact-list-with-versions.md)
+
 ## License
 
 The license for this repository is specified in [LICENSE.md](LICENSE.md)
 
+Detailed metada with licenses can be found in: [Component Governance cgmanifes.json](./cgmanifest.json)
+
 ## Building
 
 Instructions for building this repository are specified in [BUILDING.md](BUILDING.md)
 
+In depth building instructions: [./docs/build.md](./docs/build.md).
```

Comment:
Let's combine these 2 very similar documents into a single file.  We can move the combined file to `/docs/build.md` if that's preferred.

Maybe the new stuff can be added as "Advanced Scenarios" or something after the current instructions.

Reply:
> Let's combine these 2 very similar documents into a single file. We can move the combined file to `/docs/build.md` if that's preferred.

I have no preferences, just wanted to add some snippets I need when I clean term history etc

> Maybe the new stuff can be added as "Advanced Scenarios" or something after the current instructions.

Sure.